client/layout/activity-panel/style.scss

```diff
@@ -155,6 +155,20 @@
 	}
 }
 
+.woocommerce-layout__activity-panel-toggle-bubble.has-unread:after {
```

Comment:
I wonder if we should add a mixin for this, since we also have the same unread-bubble style in `ActivityCard`s (no action needed in this PR, just thinking about it)